src/GlobalErrorHandler.js

```diff
@@ -21,26 +23,65 @@ class GlobalErrorHandler extends React.Component {
     return (
       <Main>
         <In>
-          <ErrorCard
-            title="Oops."
-            detailsTitle={error.message}
-            detailsContent={errorStack}
-            supportUrl="https://github.com/aragon/aragon/issues/new"
-            showReloadButton
-          >
-            Something went wrong and the application crashed. Reloading might
-            solve the problem, or you can{' '}
-            <SafeLink href="https://github.com/aragon/aragon/issues/new">
-              create an issue
-            </SafeLink>{' '}
-            on GitHub so we can help.
-          </ErrorCard>
+          {error.name === 'DAONotFound' ? (
```

Comment:
Since we define these error types ourselves, it would be nicer to use `error instanceOf DAONotFound`.